includes/Discovery.php

```diff
@@ -163,6 +158,31 @@ protected function get_schemaorg_metadata(): array {
 		$post = get_queried_object();
 
 		if ( $post instanceof WP_Post ) {
+			$publisher_logo_id = get_post_meta( $post->ID, Story_Post_Type::PUBLISHER_LOGO_META_KEY, true );
```

Comment:
If we use a story model here, it means we can load other fields from the story model, like 
- title
- url
- poster ( maybe ).

Reply:
tried this now